vision/cloud-client/detect/beta_snippets.py

```diff
@@ -174,6 +174,99 @@ def detect_handwritten_ocr_uri(uri):
 # [END vision_handwritten_ocr_gcs_beta]
 
 
+# [START vision_fulltext_detection_pdf_beta]
+def detect_document_features(path):
+    """Detects document features in a PDF/TIFF/GIF file.
+
+    Args:
+    path: The path to the local file.
+    """
+    from google.cloud import vision_v1p4beta1 as vision
+    client = vision.ImageAnnotatorClient()
+
+    with open(path, 'rb') as pdf_file:
+        content = pdf_file.read()
+
+    # Other supported mime_types: image/tiff' or 'image/gif'
+    mime_type = 'application/pdf'
+    input_config = vision.types.InputConfig(
+        content=content, mime_type=mime_type)
```

Comment:
nit: (up to you if you want to change) we could even remove the `mime_type` variable
```
input_config = vision.types.InputConfig(
        content=content,
        mime_type='application/pdf')  # Other mime_types: image/tiff, image/gif
```

Reply:
Yeah, but it may be a good idea to keep it for clarity anyway.